Modules/Core/Support/InvoiceHelper.php

```
@@ -0,0 +1,135 @@
+ * Static helper class converted from procedural functions.
+ */
+class InvoiceHelper
+{
+    /**
+     * Returns the invoice image.
+     */
+    public static function invoice_logo(): string
+    {
+        // TODO: Migrate remaining CodeIgniter dependencies to Laravel
+    
+        if ($bridge->settings()->setting('invoice_logo')) {
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$bridge' which will lead to PHP notices. (undefined)

(UndefinedVariable)